install_config/configuring_aws.adoc

```diff
@@ -105,3 +106,22 @@ Access keys are obtained when setting up your AWS IAM user.
 ====
 
 Then, start or restart OpenShift services on the master and all nodes.
```

Comment:
This is what is essentially "applying your configuration changes." So I'd move this line down and make it the first line under the new "Applying Configuration Changes". If you actually show what services to restart, you'll have to consider the OSE vs Origin service name differences too w/ ifdefs. Something like:

```
Start or restart OpenShift services on all master and node hosts to apply your configuration changes:

ifdef::openshift-enterprise[]
----
# systemctl atomic-openshift-master restart
# systemctl atomic-openshift-node restart
----
endif::[]
ifdef::openshift-origin[]
----
# systemctl origin-master restart
# systemctl origin-node restart
----
endif::[]
```